<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>535</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>휴먼모음T</family>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
		background-color :rgb(0, 0, 0);
	color: #fff;
	font: 9pt &quot;휴먼모음T&quot;;
}
QPushButton{
    color:#e8e8e8;
    border-radius: 8px;
	background-color : #2c2e31;
}
QTextEdit{
	color: #fff;
}
</string>
  </property>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>119</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	background-color:#505459;
	color: #fff;
}
QPushButton:pressed{
	border-bottom-color: rgb(0, 0, 0);
	border-right-color: rgb(0, 0, 0);
    background-color:rgb(162, 169, 180);
}
</string>
   </property>
   <property name="text">
    <string>모델 불러오기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>119</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	background-color:#505459;
	color: #fff;
}
QPushButton:pressed{
	border-bottom-color: rgb(0, 0, 0);
	border-right-color: rgb(0, 0, 0);
    background-color:rgb(162, 169, 180);
}
</string>
   </property>
   <property name="text">
    <string>데이터셋 폴더선택</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>119</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	background-color:#505459;
	color: #fff;
}
QPushButton:pressed{
	border-bottom-color: rgb(0, 0, 0);
	border-right-color: rgb(0, 0, 0);
    background-color:rgb(162, 169, 180);
}
</string>
   </property>
   <property name="text">
    <string>데이터셋 적용폴더</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>119</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	background-color:#505459;
	color: #fff;
}
QPushButton:pressed{
	border-bottom-color: rgb(0, 0, 0);
	border-right-color: rgb(0, 0, 0);
    background-color:rgb(162, 169, 180);
}
</string>
   </property>
   <property name="text">
    <string>텍스트 검출</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>119</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	background-color:#505459;
	color: #fff;
}
QPushButton:pressed{
	border-bottom-color: rgb(0, 0, 0);
	border-right-color: rgb(0, 0, 0);
    background-color:rgb(162, 169, 180);
}
</string>
   </property>
   <property name="text">
    <string>이미지 검출</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>551</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="original_file">
    <attribute name="title">
     <string>Original File</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>551</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 1px;
border-color: #rgb(0, 0, 0);
background-color : #2c2e31;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="result_file">
    <attribute name="title">
     <string>Result File</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>551</width>
       <height>491</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 1px;
border-color: #rgb(0, 0, 0);
background-color : #2c2e31;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="chart">
    <attribute name="title">
     <string>Chart</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>551</width>
       <height>491</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 1px;
border-color: #rgb(0, 0, 0);
background-color : #2c2e31;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>211</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="original_dir">
    <attribute name="title">
     <string>Original Dir</string>
    </attribute>
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>211</width>
       <height>381</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-style: solid;
border-width: 1px;
border-color: #rgb(0, 0, 0);
</string>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="result_dir">
    <attribute name="title">
     <string>Result Dir</string>
    </attribute>
    <widget class="QTreeView" name="treeView_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>211</width>
       <height>381</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-style: solid;
border-width: 1px;
border-color: #rgb(0, 0, 0);
</string>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>430</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>결과확인</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{
	background-color:#505459;
	color: #fff;
}
QPushButton:pressed{
	border-bottom-color: rgb(0, 0, 0);
	border-right-color: rgb(0, 0, 0);
    background-color:rgb(162, 169, 180);
}
</string>
     </property>
     <property name="text">
      <string>차트로 확인</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{
	background-color:#505459;
	color: #fff;
}
QPushButton:pressed{
	border-bottom-color: rgb(0, 0, 0);
	border-right-color: rgb(0, 0, 0);
    background-color:rgb(162, 169, 180);
}
</string>
     </property>
     <property name="text">
      <string>뭐였</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
